### PR TITLE
container feature related

### PR DESCRIPTION
**Pull request by Eskibear**
upgrade to azure sdk v1.1.2
container web app on linux workflow.

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.container/src/com/microsoft/azuretools/container/presenters/StepTwoPagePresenter.java [hunk 1]
```diff
@@ -22,34 +22,208 @@
...
+     * @param index
+     *            subscription index on selection
+     */
+    public void onChangeSubscription(int index) {
+        doFetchResourceGroup(binderSubscriptionDetails.get(index));
+        getMvpView().fillResourceGroups(binderResourceGroup);
+    }
+
+    /**
+     * used for refreshing list.
```
Comment (adashen) [inline PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.container/src/com/microsoft/azuretools/container/presenters/StepTwoPagePresenter.java]: null check for the return of observable

File: PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.container/src/com/microsoft/azuretools/container/presenters/StepTwoPagePresenter.java [hunk 2]
```diff
@@ -22,34 +22,208 @@
...
+     */
+    public void onRefreshWebAppsOnLinux() {
+        getMvpView().onRequestPending(TEXT_LISTING_AEB_APP_ON_LINUX);
+        Observable.fromCallable(() -> {
+            updateWebAppOnLinuxList();
+            return null;
+        }).subscribeOn(Schedulers.io()).subscribe(res -> {
```
Comment (adashen) [inline PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.container/src/com/microsoft/azuretools/container/presenters/StepTwoPagePresenter.java]: use scheduler provider

File: PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.container/src/com/microsoft/azuretools/container/ui/wizard/publish/StepOnePage.java [hunk 3]
```diff
@@ -22,55 +22,68 @@
...
+        super.dispose();
     }
 
-    // View Actions
+    // com.microsoft.azuretools.core.mvp.ui.base.MvpView
     @Override
-    public void setCompleteStatus(boolean flag) {
-        setPageComplete(flag);
+    public void onErrorWithException(String message, Exception ex) {
+        ConsoleLogger.error(String.format("%s\n%s", message, ex.getMessage()));
```
Comment (adashen) [inline PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.container/src/com/microsoft/azuretools/container/ui/wizard/publish/StepOnePage.java]: thread issue?

File: Utils/pom.xml [hunk 4]
```diff
@@ -270,7 +270,7 @@
             <dependency>
                 <groupId>com.microsoft.azure</groupId>
                 <artifactId>azure</artifactId>
-                <version>1.1.0</version>
+                <version>1.1.2</version>
```
Comment (adashen) [inline Utils/pom.xml]: any impact for add lib feature in eclipse / intelliJ for the version update?

File: PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.container/src/com/microsoft/azuretools/container/presenters/StepOnePagePresenter.java [hunk 5]
```diff
@@ -62,23 +68,20 @@ public void progress(ProgressMessage message) throws DockerException {
             }).subscribeOn(Schedulers.io()).subscribe(wal -> {
                 // persist registry information
                 DefaultLoader.getIdeHelper().invokeAndWait(() -> {
-                    doUpdateRuntimeRegistryInfo(registryUrl, registryUsername, registryPassword);
+                    updateRuntimeRegistryInfo(registryUrl, registryUsername, registryPassword);
                     V v = getMvpView();
                     if (v != null) {
```
Comment (jdneo) [inline PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.container/src/com/microsoft/azuretools/container/presenters/StepOnePagePresenter.java]: How about using 'isViewDetached()' in MvpPresenter

File: PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.container/src/com/microsoft/azuretools/container/ui/wizard/publish/StepOnePage.java (hunk 3 above)
Comment (Eskibear) [inline PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.container/src/com/microsoft/azuretools/container/ui/wizard/publish/StepOnePage.java]: no

File: Utils/pom.xml (hunk 4 above)
Comment (Eskibear) [inline Utils/pom.xml]: no

File: PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.container/src/com/microsoft/azuretools/container/presenters/StepTwoPagePresenter.java (hunk 1 above)
Comment (Eskibear) [inline PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.container/src/com/microsoft/azuretools/container/presenters/StepTwoPagePresenter.java]: checked

**Pull request merged**